gradio/wasm_utils.py

```diff
@@ -0,0 +1,20 @@
+import sys
+
+# See https://pyodide.org/en/stable/usage/faq.html#how-to-detect-that-code-is-run-with-pyodide
+is_wasm = sys.platform == "emscripten"
+is_pyodide = "pyodide" in sys.modules
```

Comment:
Suggest naming these as constants
```suggestion
IS_WASM = sys.platform == "emscripten"
IS_PYIODIDE = "pyodide" in sys.modules
```

Reply:
Also, is `is_pyiodide` used at all?

Reply:
👍